ecllibrary/std/DataPatterns/Profile.ecl

```diff
@@ -1442,14 +1529,18 @@ EXPORT Profile(inFile,
             #IF(%'dsNameValue'% != '')
                 #SET(numValue, REGEXFIND('^(\\d+):', %'dsNameValue'%, 1))
                 #SET(nameValue, REGEXFIND(':([^:]+)$', %'dsNameValue'%, 1))
-                + _Profile
+                // The child dataset should have been extracted into its own
+                // local attribute; reference it during our call to the inner
+                // profile function macro
+                #SET(temp, #MANGLE(%'nameValue'%))
```

Comment:
Diff: Use previously-created local attribute, referencing a child dataset, to work around ECL problems with dot-syntax referencing in some circumstances